#!/usr/bin/env python
PACKAGE = "object_psd"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("leaf_size", double_t, 0, "Leaf Size for VoxelGrid", 0.01, 0.001, 0.1)
gen.add("ransac_it", int_t, 0, "RANSAC Iterations", 1000, 100, 10000)
gen.add("sphere_th", double_t, 0, "Sphere Threshold", 0.01, 0.005, 1.0)
gen.add("cylinder_th", double_t, 0, "Cylinder Threshold", 0.01, 0.005, 1.0)

exit(gen.generate(PACKAGE, "object_psd", "PSD"))